### PR TITLE
pgbench: add page

### PR DESCRIPTION
**Pull request by andyatkinson**
Primarily based on this CloudBees blog post:
<https://www.cloudbees.com/blog/tuning-postgresql-with-pgbench/>

And the PG 10 pgbench documentation:
<https://www.postgresql.org/docs/10/pgbench.html>

- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/pgbench.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# pgbench
+
+> Run a benchmark test on PostgreSQL.
+> More information: <https://www.postgresql.org/docs/10/pgbench.html>.
+
+- Initialize database example with scaling option of 50 times the default size:
+
+`pgbench -i -s 50 example`
```
Comment (bl-ue) [inline pages/common/pgbench.md:8]:
```suggestion
`pgbench --initialize --scale={{50}} {{database_name}}`
```

**Review by bl-ue — CHANGES_REQUESTED** [review]
Thanks for the new page @andyatkinson!  I've left some comments below for you to review.

In general:
1. We use long options in examples when possible
2. User-entered flag/argument values get wrapped in our custom token syntax (`{{` and `}}`)
3. We try to use more generic names for objects, e.g. `path/to/file` and `database_name`, rather than `myfile.txt` or `example`.

**Comment by andyatkinson** [comment]
@bl-ue Those are excellent suggestions. This project had great contribution guidelines and workflow around those contributions. The long options I read about in the guidelines but wasn't sure if they applied here, although seeing them written out as the suggestions, I like them better too.

I somehow missed the custom token syntax, but that's great. 

Using `database_name` also makes sense here as it communicates more to the reader than `example`. I actually was confused myself using `pgbench` the first time running the command, and realizing I needed to first create a database named `example`.

File: pages/common/pgbench.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# pgbench
+
+> Run a benchmark test on PostgreSQL.
+> More information: <https://www.postgresql.org/docs/10/pgbench.html>.
+
+- Initialize database example with scaling option of 50 times the default size:
```
Comment (bl-ue) [inline pages/common/pgbench.md:6]:
```suggestion
- Initialize a database with scaling option of 50 times the default size:
```

**Review by marchersimon — APPROVED** [review]
Looks good to me

**Pull request merged**